api/src/main/java/edu/cornell/mannlib/vitro/webapp/email/FreemarkerEmailFactory.java

```diff
@@ -111,7 +127,7 @@ InternetAddress getReplyToAddress() {
 	Session getEmailSession() {
```

Comment:
I suppose here should be a private access modifier. I suppose friendly (not specified) access modifier has been used for getSmtpHost() and getReplyToAddress(), because those methods are invoked by inner Setup class. It might be it was mandatory in the past for some Java version, although I think it is not case now, I think that inner class can access all attribute of outer class. However, I don't see why getEmailSession() method should be defined without access modifier.

Reply:
Sure.